os/fs/romfs/fs_romfs.c

```diff
@@ -1040,6 +1100,42 @@ static int romfs_statfs(FAR struct inode *mountpt, FAR struct statfs *buf)
 }
 
 /****************************************************************************
+ * Name: romfs_stat_common
+ *
+ * Description:
+ *   Return information about a file or directory
```

Comment:
Kindly add details about input args and o/p

Reply:
Done.